problems/removeStringsEndingInBang.js

```diff
@@ -11,8 +11,14 @@
 * returns: ["", "a"]
 */
 
-function removeStringsEndingInBang() {
-
+const removeStringsEndingInBang = (arr) => {
+    let newArr = [];
+    for(let i = 0; i < arr.length; i++){
+        if(!arr[i].includes("!")){
+            newArr.push(arr[i]);
+        }
```

Comment:
This if statement does not check if the string **ends** with `!` it only checks if it is not included anywhere in the string